src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    static final int NUMBER = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        for (int i = 0; i <= NUMBER; i++) {
+            Ball ball = new Lottery().getRandomBall();
```

Comment:
Do we really need to create new `Lottery` instance during each iteration? Feel free to move `Lottery` creation out of the loop scope. 
Sorry for confusion, [this](https://github.com/mate-academy/jv-lottery/pull/1172#discussion_r1339603374) comment referred to magic numbers in the loop, which you replaced with `NUMBER` constant.